controllers/get-users.controller.js

```diff
@@ -0,0 +1,13 @@
+const { selectUsers } = require("../models/models")
+
+const getUsers = (req, res, next) => {
+    selectUsers()
+    .then((users) => {
+    res.status(200).send(users)
+    })
+    .catch((err) => {
+        next(err)
+    })
```

Comment:
We should not be sending users directly. We should be sending an object with a descriptive key holding the users array we want to send